tests/adapter/mongo/test_case_handling.py

```diff
@@ -107,11 +108,11 @@ def test_get_cases_no_synopsis(real_adapter, case_obj, institute_obj, user_obj):
 
     adapter = real_adapter
     # GIVEN a real database with no cases
-    assert real_adapter.cases().count() == 0
+    assert sum(1 for i in adapter.cases()) == 0
```

Comment:
Isn't it more convenient to use assert find_one here as well or is False in this case?

Reply:
I didn't have the energy to update all files in that way. Sometimes I just used find/replace 😄